core/commands/swarm.go

```diff
@@ -201,12 +232,31 @@ var swarmPeeringRmCmd = &cmds.Command{
 				return err
 			}
 		}
+
+		save, _ := req.Options[swarmSaveOptionName].(bool)
```

Comment:
can we check here if the casting is not ok to throw a descriptive error to the user?

Reply:
Testing this as is a couple ways seems to indicate that this issue is or works as expected:
```
$ ipfs swarm peering rm ${peeringID} --save "true"
Error: failed to parse peer ID: illegal base32 data at input byte 0

$ ipfs swarm peering rm ${peeringID} --save="true"
# works fine

$ ipfs swarm peering rm ${peeringID} --save true
Error: failed to parse peer ID: illegal base32 data at input byte 0

$ ipfs swarm peering rm ${peeringID} --save=true
# works fine
```